hadoop-hdds/common/src/main/java/org/apache/hadoop/util/OzoneUtils.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership.  The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.hadoop.util;
+
+import com.google.protobuf.ServiceException;
+import org.apache.hadoop.ipc.RPC;
+import org.apache.hadoop.ipc.RemoteException;
+import org.apache.hadoop.ipc.RpcException;
+import org.apache.hadoop.ipc.RpcNoSuchMethodException;
+import org.apache.hadoop.ipc.RpcNoSuchProtocolException;
+import org.apache.hadoop.security.AccessControlException;
+import org.apache.hadoop.security.token.SecretManager;
+
+/**
+ * Common Util class for SCM, OM and other components.
+ */
+public final class OzoneUtils {
```

Comment:
Nit: can we add this new method to the existing `HddsUtils` class instead of creating a new one?

https://github.com/apache/ozone/blob/c088cd1d96895d8f6a13fa3903909ea5b85ceb04/hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsUtils.java#L78

Note, we also already have `OzoneUtils`, but it's in `ozone-common`:

https://github.com/apache/ozone/blob/c088cd1d96895d8f6a13fa3903909ea5b85ceb04/hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/web/utils/OzoneUtils.java#L44